internal/pkg/cli/dialog/host.go

```diff
@@ -10,24 +10,27 @@ import (
 	"github.com/keboola/keboola-as-code/internal/pkg/utils/strhelper"
 )
 
-var ErrMissingStorageApiHost = fmt.Errorf(`missing Storage API host`)
+var ErrMissingStorageApiHost = fmt.Errorf("missing Storage API host")
 
-func (p *Dialogs) AskStorageApiHost(options *options.Options) (string, error) {
-	host := options.GetString(`storage-api-host`)
+func (p *Dialogs) AskStorageApiHost(d hostAndTokenDependencies) (string, error) {
+	opts := d.Options()
+	host := opts.GetString("storage-api-host")
 	if len(host) == 0 {
 		host, _ = p.Ask(&prompt.Question{
 			Label:       "API host",
 			Description: "Please enter Keboola Storage API host, eg. \"connection.keboola.com\".",
 			Validator:   StorageApiHostValidator,
 		})
+	} else if opts.KeySetBy("storage-api-host") == options.SetByEnv {
+		d.Logger().Infof(`Storage API host "%s" set from ENV.`, host)
```

Comment:
Drobnost, rovno som pridal hlasku, ze ak je nastaveny host/token cez ENV, tak to vypiseme.
Vtedy dialog nezobrazujeme a bola staznost, ze user nevie odkial sa hodnota berie.
Ked ju zada cez dialog/flag, tak je to jasne, problem je len ENV.